@kiva/kv-components/src/vue/KvCheckbox.vue

```diff
@@ -162,10 +211,8 @@ export default {
 
 		const setChecked = () => {
 			if (Array.isArray(modelValue.value)) {
-				// if the model is array like <kv-checkbox v-model="['item1', 'item2']" value="item1">
 				isChecked.value = modelValue.value.includes(value.value);
 			} else {
-				// else it's a boolean like <kv-checkbox v-model="true">
```

Comment:
These are helpful, why were they removed?

Reply:
The second comment is useful too as it describes the difference between passing an array or bool to v-model